user-service/model/user-model.js

```diff
@@ -26,6 +31,11 @@ const UserModelSchema = new Schema({
     required: true,
     default: false,
   },
+  isDarkMode: {
+    type: Boolean,
+    required: true,
+    default: false,
+  }
```

Comment:
Not very important but if we're porting in the whole theme switching into the backend, do we want to include the "systems" option?

alternatively, we don't have to store the darkmode preference on the backend and make it a frontend thing by storing it in localstorage

Reply:
I think let's remove the dark mode preference from the backend. Will refactor the backend to remove the isDarkMode field